### ISSUE TITLE
Weird Error

### ISSUE THREAD

**Issue by PaladinFallen** [Sideways-Sky/MultiMine#1]
[18:01:23] [Server thread/ERROR]: Could not pass event PlayerArmSwingEvent to MultiMine v1.1
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100) ~[?:?]
	at jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106) ~[?:?]
	at jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302) ~[?:?]
	at java.util.Objects.checkIndex(Objects.java:385) ~[?:?]
	at java.util.ArrayList.get(ArrayList.java:427) ~[?:?]
	at net.sideways_sky.multimine.DamagedBlock.delete(DamagedBlock.java:54) ~[MultiMine-1.1.jar:?]
	at net.sideways_sky.multimine.DamagedBlock.damageTick(DamagedBlock.java:42) ~[MultiMine-1.1.jar:?]
	at net.sideways_sky.multimine.Events.onPlayerArmSwing(Events.java:39) ~[MultiMine-1.1.jar:?]
	at com.destroystokyo.paper.event.executor.StaticMethodHandleEventExecutor.execute(StaticMethodHandleEventExecutor.java:39) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:git-Paper-408]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.20.4.jar:git-Paper-408]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[paper-1.20.4.jar:git-Paper-408]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleAnimate(ServerGamePacketListenerImpl.java:2506) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundSwingPacket.handle(ServerboundSwingPacket.java:25) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundSwingPacket.a(ServerboundSwingPacket.java:11) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:54) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.20.4.jar:git-Paper-408]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1455) ~[paper-1.20.4.jar:git-Paper-408]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:194) ~[paper-1.20.4.jar:git-Paper-408]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1432) ~[paper-1.20.4.jar:git-Paper-408]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1355) ~[paper-1.20.4.jar:git-Paper-408]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1333) ~[paper-1.20.4.jar:git-Paper-408]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1222) ~[paper-1.20.4.jar:git-Paper-408]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[paper-1.20.4.jar:git-Paper-408]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]

**Comment by Sideways-Sky** [comment]
Thanks for the issue! Could I get the following info?
- Debug info surrounding the error (set debug to true in config and reproduce the error)
- Steps to reproduce, if you know (or what you were doing when the error occurred)

Thanks!

**Comment by PaladinFallen** [comment]
Debug info below:
https://pastebin.com/BuqnCE1m

Steps to reproduce: It's random, but the only thing me and another player was doing, was just mining..

**Comment by Threeleggedog68** [comment]
I invested into this miner I have all my emails to support when they made a mistake by saying my ID was a pass port. They wouldn't let ne correct it. I had the app on my phone for the longest time and I mined alot if bitcoin. I could just be setting there and it would just disappear. I don't value believe they started the miner to rip people of bur they started taking my bitcoin that I had earn. And I gave them a $140 total.

**Comment by Threeleggedog68** [comment]
I invested into this miner I have all my emails to support when they made a mistake by saying my ID was a pass port. They wouldn't let ne correct it. I had the app on my phone for the longest time and I mined alot if bitcoin. I could just be setting there and it would just disappear. I don't value believe they started the miner to rip people of bur they started taking my bitcoin that I had earn. And I gave them a $140 total.

**Comment by Threeleggedog68** [comment]
Why you delete my comments when I tell the truth

**Comment by Sideways-Sky** [comment]
Because it wasn't related. I think you might have the wrong repository. This is a plugin for minecraft servers. What you described in your comments was what I believe to be related to mining cryptocurreency.

**Comment by Threeleggedog68** [comment]
I don't mean any disrespect to anyone to my knowledge that is. I don't quite understand my experience with what happened. It was a total nightmare and when I saw they had an account with git hub I was upset from the loss I had. I invested and watched my bitcoin accumulate and they just took it and said it was my fault. I would rather have the bitcoin my account earned before they took it and it was alot. And now I can't get a refund from Google for my inapp purchase a app I downloaded I got from Google play.
